mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,283 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Settings'
+description: |
+  gcloud iap settings - manage IAP settings
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Customizing IAP': 'https://cloud.google.com/iap/docs/customizing'
+  api: 'https://cloud.google.com/iap/docs/reference/rest/v1/IapSettings'
+base_url: '{{name}}:iapSettings'
+self_link: '{{name}}:iapSettings'
+create_verb: :PATCH
+update_verb: :PATCH
+update_url: '{{name}}:iapSettings'
+import_format: ['{{name}}/iapSettings']
+id_format: '{{name}}/iapSettings'
+# Skipping the sweeper because instances will be deleted during cluster sweeps
+skip_sweeper: true
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    diff_suppress_func: 'NameDiffSuppress'
+    description: |
```

Comment:
The `diff_suppress_func` is not prefect and also suppresses the diffs between two resources. `ignore_read: true` can be used instead.

Reply:
So diff_suppress_func and custom code (constant) can be removed and replaced with `ignore_read : true`?

Reply:
Right.